mne/io/egi/egi.py

```diff
@@ -152,26 +157,49 @@ def read_raw_egi(
     Notes
     -----
     The trigger channel names are based on the arbitrary user dependent event
-    codes used. However this function will attempt to generate a **synthetic
-    trigger channel** named ``STI 014`` in accordance with the general
-    Neuromag / MNE naming pattern.
-
-    The event_id assignment equals ``np.arange(n_events) + 1``. The resulting
-    ``event_id`` mapping is stored as attribute to the resulting raw object but
-    will be ignored when saving to a fiff. Note. The trigger channel is
+    codes used. However, when ``events_as_annotations=False``,
+    this function will attempt to generate a **synthetic trigger channel**
+    named ``STI 014``. The event_id assignment equals ``np.arange(n_events) + 1``.
+    The resulting ``event_id`` mapping is stored as attribute to the resulting raw
+    object but will be ignored when saving to disk. The trigger channel is
     artificially constructed based on timestamps received by the Netstation.
-    As a consequence, triggers have only short durations.
+    As a consequence, triggers have only short durations. This step will also fail
+    if events are not mutually exclusive.
 
-    This step will fail if events are not mutually exclusive.
+    For these reasons, it is recommended to use ``events_as_annotations=True``.
```

Comment:
I recommend reorganizing the Notes section, to put the info in this order:

1. what happens when `events_as_annotations=True` (put this first because it's the recommended way, and easier to explain)
2. what happens when `events_as_annotations=False`
    1. new stim channel STI014 created (w/ crossref to glossary entry for stim chans)...
    2. ...that contains brief (1-sample?) pulses where the Netstation file had event timestamps.
    3. An `event_id` dictionary is attached (where?) to the Raw object...
    4. ...that will have arbitrary sequential integer IDs for the events...
    5. ...will fail if any timestamps are duplicated (is that the right way to state the failure mode?)...
    6. ...and will not survive a save/load roundtrip.